### ISSUE TITLE
I want to change rainbowminer complete language?

### ISSUE THREAD

**Issue by MTCHANNELL** [RainbowMiner/RainbowMiner#2550]
Hello. Although English is a language spoken worldwide. Unfortunately, there are people like me who cannot understand. Frankly, constantly translating is tiring. I speak Turkish as a language. Do you have a chance to publish the sections I translated for Turkish-speaking users on your github page?

What stage am I at?

01-) setup.bat (questions asked during the first installation) (0% translated. It will be fully translated into Turkish at the appropriate time.)

02-) webgui (99% translated into Turkish. It will be fully translated into Turkish at the appropriate time.)

If I share the files with you after the Turkish translation process is completed, would you have a chance to publish it, sir?

Your software is excellent and I really enjoy using it. I learn new things every day from your program.

sample image;
![Adsız](https://github.com/RainbowMiner/RainbowMiner/assets/94814602/12ed1ca0-ddec-4b69-93e4-5cfd2cab103d)

**Comment by Robbylikes** [comment]
The best option for you is to copy and paste the text from the ui into a translator.

sizin için en iyi seçenek, metni kullanıcı arayüzünden bir çevirmene kopyalayıp yapıştırmaktır.

**Comment by RainbowMiner** [comment]
Thank you very much! This is great. I will think about multi language, so that it becomes easy to maintain.

**Comment by MTCHANNELL** [comment]
I completed the translation into Turkish. I will share with you the files I translated when I get home from work. Thank you for your attention.

**Comment by MTCHANNELL** [comment]
I think I've finally finished it. I wanted to share it with you.

the changes that have been made;

00-) What has been changed in "setup.bat"?                                                                                   
     It has been translated into Turkish at a rate of 99.9%.(Some words will be corrected again.)                            
     Coloring was done in a way worthy of the name "rainbowminer"                                                            
     A space was added between each question. (for easier reading)                                                           
     Added "open as administrator/fixed window mode/solution to vowel problem" codes to the start lines of all ".bat" files. 

01-) What has been changed in "webgui"?
     It has been translated into Turkish at a rate of 99.9%.(Some words will be corrected again.)
     An animated gif of the "Rainbowminer" text on the upper left was prepared. The text "rainbowminer" is now dancing
     The "web gui" logo has been replaced with a clearer and more vibrant but identical logo
     Turkish flag logo was enlarged. Those who wish can put their own country flag in a larger size by following the path "web\images\flags"
     
in short;
The articles were translated into Turkish
The logo and images were replaced with higher quality ones without damaging their originality.     

So which files did I use to translate to Turkish?
00-) Core.psm1
01-) Setup.psm1
02-) "web" folder

I am compressing all the changes I made and sharing them with you
download link: 

setup.bat (video valid for 2 days)
https://streamable.com/br963r

webgui video: (video valid for 2 days)
https://streamable.com/m26y7i
